ts/src/bitopro.ts

```diff
@@ -767,18 +767,22 @@ export default class bitopro extends Exchange {
         // we need to have a limit argument because "to" and "from" are required
         if (limit === undefined) {
```

Comment:
I think something was broken here, this was working 
```
n bitopro fetchOHLCV "BTC/USDT" 1h undefined 1 --request
2024-03-29T11:49:14.733Z
Node.js: v18.18.0
CCXT v4.2.85
bitopro.fetchOHLCV (BTC/USDT, 1h, , 1)
BadRequest bitopro {"error":"Wrong parameter: from"}
---------------------------------------------------
[BadRequest] bitopro {"error":"Wrong parameter: from"}

    at throwBroadlyMatchedException  Users/cjg/Git/ccxt10/ccxt/js/src/base/Exchange.js:4055  
    at handleErrors                  Users/cjg/Git/ccxt10/ccxt/js/src/bitopro.js:1721        
    at                               Users/cjg/Git/ccxt10/ccxt/js/src/base/Exchange.js:971   
    at processTicksAndRejections     node:internal/process/task_queues:95                    
    at fetch2                        Users/cjg/Git/ccxt10/ccxt/js/src/base/Exchange.js:3617  
    at request                       Users/cjg/Git/ccxt10/ccxt/js/src/base/Exchange.js:3620  
    at fetchOHLCV                    Users/cjg/Git/ccxt10/ccxt/js/src/bitopro.js:773         
    at async run                     Users/cjg/Git/ccxt10/ccxt/examples/js/cli.js:345        

bitopro {"error":"Wrong parameter: from"}
```

Reply:
oh, what a silly mistake i had. fixed.